### ISSUE TITLE
Restructuring "Associated barcodes" and "Barcodes not entered into App" in Issues log

### ISSUE THREAD

**Issue by RebekkaML** [NHMDenmark/Herbarium-Sheets-workstation#101]
In the issues log, we currently have one tab for barcodes, that haven't been entered into the DigiApp yet (e.g. when we find a sheet without barcode during imaging at the Herbarium) and another tab for barcodes that need to be associated as an MSO.

However, these issues often belong together. The most common case for this is when the person imaging at the Herbarium notices that an MSO was missed and assigns it another barcode. This barcode is then 1) not in the DigiApp yet AND 2) needs to be associated with another barcode.

Therefore, it would make sense to combine these 2 tabs into 1.

Furthermore, the process of entering data into the "not entered into App"-sheet takes some time at the moment. We enter all the information that we need to create record with the DigiApp later (Box, type, preparation, geographic region, taxon name etc.). 
Pip suggested, that we instead enter another barcode that has identical information and then copy it from that one when entering the missing barcode into the app.

I will make a suggestion of how the new table could look like, and then discuss it at a Digitizer meeting.

**Comment by RebekkaML** [comment]
There are some possible issues with using another barcode to copy information from:

For missed MSOs it is easy, we can just add the barcode that is already on the sheet. For other sheets, we could pick the barcode of the sheet right before or after it, or any sheet from the same folder. The only problem is if the sheet is the only one in a folder, then there might not be an identical barcode we can use as a reference.

Also, we would need to manually enter the reference barcode at the imaging station, which might cause errors. This would make it difficult to find the right information when entering the missed barcode into the App later. Or we need a barcode reader at the imaging station as well, but that takes up space and we would not need it often.

**Comment by RebekkaML** [comment]
Made a new table for missed barcodes with the option to mark them as associated. This needs to be discussed with the other digitizers before implementing it.

[New missed barcodes table.xlsx](https://github.com/user-attachments/files/15910930/New.missed.barcodes.table.xlsx)

**Comment by RebekkaML** [comment]
I sent the new missed barcodes table to the Digitization team for feedback today.

**Comment by RebekkaML** [comment]
Collected feedback, mostly positive. 

One comment by Astrid: "If a MSO is missed and we do not resolve this using the Digi App, you will need another way to get the containerID for the association (and you will then need to add it to the reference record before using it to create the new record). Do we have a protocol in place for getting containerIDs without the Digi App?
I have discussed this with Fedor and he said that the container ID can be any unique string of characters and we can just use a random number generator for this. 
Alternatively, we can use the Digi App to generate the container number so it always has the same format.
During the digitizer meeting today, Pip suggested to agree on a format that includes e.g. the barcode number and the date to insure it is unique, but we didn't decide on anything specific yet.

Everyone agreed the new table is good to be implemented now. I transferred any unsolved issues from the old tables to the new one, made safety copies and then deleted them. The new table is now a tab in the issues log called "Barcodes and Assoc. not in App". 

This also needs to be added to the next version of the Imaging guide, I added it to that ticket as well (#102).
